app/assets/stylesheets/sass/buttons.scss

```diff
@@ -43,3 +43,9 @@ input.update-comment-crud-buttons.tiny {
     @include buttonhovers(purple);
   }
 }
+input.search-crud-buttons {
+  @include buttonstyles(#333);
+  &:hover{
+    @include buttonhovers(purple);
```

Comment:
Color `purple` should be written in hexadecimal form as `#800080`<br>Color literals like `purple` should only be used in variable declarations; they should be referred to via variable everywhere else.